### PR TITLE
Feat: Translation in French of first part Manuel Three.js

### PR DESCRIPTION
**Pull request by David-Moisan**
Related issue: #18902

**Description**

This is a translation in french of first par Manuel Three.js

### CODE REVIEW

**Comment by mrdoob** [comment]
@greggman does this look good to you?

**Comment by greggman** [comment]
Well, um, I don't read French 😭. Things that stick out

(1) These don't seem to be related to threejsfundamentals so I'm not sure what they are supposed to be like
(2) Code indenting looks off in a few places
(3) filenames are in French. Not sure if that's what you want. I tended to keep the filenames the same so urls only change the language code

**Comment by David-Moisan** [comment]
Oh sorry for the name of files, i change this soon as possible ^^.

**Comment by Mugen87** [comment]
Sorry, it seems this PR got lost.

It seems the PR misses two changes in `docs/index.html` so it's possible to select French language. Check out #24455 for more details.

File: docs/manual/fr/introduction/Bibliothèques-et-plugins.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Mugen87) [inline docs/manual/fr/introduction/Bibliothèques-et-plugins.html:2]:
```
<html lang="fr">
```

**Comment by Mugen87** [comment]
Closing in favor of #24516.

**Pull request closed without merging**